inference-engine/src/transformations/include/transformations/rt_info/old_api_map_attribute.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Defines old API map attribute
+ * @file old_api_map_attribute.hpp
+ */
+
+#pragma once
+
+#include <assert.h>
+
+#include <functional>
+#include <memory>
+#include <ngraph/attribute_visitor.hpp>
+#include <ngraph/node.hpp>
+#include <ngraph/variant.hpp>
+#include <openvino/core/rtti.hpp>
+#include <set>
+#include <string>
+#include <transformations_visibility.hpp>
+#include <utility>
+
+namespace ov {
+
+class OldApiMap;
+/**
+ * @ingroup ie_runtime_attr_api
+ * @brief OldApiMapAttr class stores the value of OldApiMap class.
+ */
```

Comment:
Please describe the usage of this attribute. What m_order and m_type means and how they can be used.

Reply:
Added comment.